src/main/fc/core.h

```diff
@@ -47,6 +47,19 @@ typedef enum {
     LAUNCH_CONTROL_MODE_COUNT // must be the last element
 } launchControlMode_e;
 
+typedef enum {
+    DISARM_REASON_SYSTEM            = 0,
```

Comment:
What is this used for?

Reply:
tests only currently.

Reply:
Not sure we get value out of having enum values that are used for tests only - probably better (i.e. resulting in more reality-like tests) to select one of the existing ones and use this in tests. If not then this enum value should be conditional to `UNIT_TEST`.

Reply:
Yeah, I was on the fence regarding it.

choose one:
a) delete it.
b) rename it to '..._RESERVED'
c) gate it for tests only.
d) leave it as-is.

Note: If I change/remove it I also have to update the PR for the viewer.

My preference is D.

Reply:
I'm all for a) - see my comment above about lack of value.

Reply:
I'm picking d)

reasoning:
* if you remove it the values you use in the tests won't make any sense.
* if you gate it the sequence of enum values doesn't start at 0, 0 should be a sane default for any structure containing the value that has be zeros.
* index into array of strings in client code should start at 0, otherwise you have to do a `-1` everywhere you index into the matching list of strings.